src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java

```diff
@@ -686,4 +686,27 @@ public void checkDefaultPagination() {
             Assert.fail("numero di default delle notifiche visualizzate non corretto");
         }
     }
+
+    public boolean controlloEsistenzaStato() {
+        this.statoNotificaField.click();
+        try {
+            List<WebElement> statiNotifica = this.elements(By.xpath("//li[@data-value]"));
+            this.getWebDriverWait(10).withMessage("Il menu a tendina dello stato notifica del filtro non è visibile").until(ExpectedConditions.visibilityOfAllElements(statiNotifica));
+            ArrayList<String> stati = new ArrayList<>(List.of("Tutti gli stati", "Depositata", "Invio in corso", "Consegnata", "Perfezionata per decorrenza termini", "Avvenuto accesso", "Annullata", "Destinatario irreperibile"));
```

Comment:
is possible passes with cucumber?

Reply:
Why would you pass something when the elements are fixed and built as such?

Reply:
so that we can modify the feature file and not the code, by doing so we are aligned with all the other tests